assembl/__init__.py

```diff
@@ -91,8 +92,9 @@ def main(global_config, **settings):
         from .lib.migration import bootstrap_db_data
         bootstrap_db_data(session, False)
 
+    config.add_static_view('react', 'react', cache_max_age=3600)
```

Comment:
Suggestions to replace "react":
- static_v2
- frontend
- assembl (but the occurence will be present 3 times)